parser/src/class/parser.rs

```diff
@@ -1,23 +1,20 @@
 use flora_syntax::ClassDeclaration;
-use nom::types::CompleteStr as Input;
-use nom::*;
-
-named!(
-    pub class_declaration<Input, ClassDeclaration>,
-    do_parse!(
-        multispace0 >>
-        tag!("class") >>
-        multispace1 >>
-        name: map!(
-            alphanumeric1,
-            |Input(name)| name
-        ) >>
-        multispace0 >>
-        char!(';') >>
-        multispace0 >>
-        (ClassDeclaration { name })
-    )
-);
+use nom::{
+    bytes::complete::tag,
+    character::complete::{alphanumeric1, char, multispace0, multispace1},
+    IResult,
+    sequence::tuple
+};
+
+
+pub fn class_declaration(input: &str) -> IResult<&str, ClassDeclaration> {
+    let (i, (_ , _, _, name, _ , _, _)) = tuple((
```

Comment:
Not sure this is more readable than before, might be better to write it something like [this](https://github.com/Geal/nomfun/blob/b6e75dc812ac3639760d5d8c8f67297a4bfb048e/benches/http.rs#L93-L102)?

Reply:
Right, your example is more readable and makes it easier to keep track of the ignored return values